### ISSUE TITLE
Getting stuck running code... Help!

### ISSUE THREAD

**Issue by nmsommer** [RobRomijnders/cnn_music#2]
Hi!

I am very interested in your code since it is set up to take in .csv files for a CNN.  I wanted to test it out and I am running in to some issues.  Could you let me know what I am doing incorrectly?  Thanks!

![cnnmusicproblem](https://cloud.githubusercontent.com/assets/24843042/22094257/2d308f04-ddda-11e6-9ea2-d267c09652c9.PNG)

**Comment by RobRomijnders** [comment]
This project was coded with Tensorflow 0.6, I believe. After that, they changed the batch norm implementation

**Comment by nmsommer** [comment]
Thanks for your reply.  Could you advise about how I could fix this?
